tests/migrations/test_operations.py

```diff
@@ -1910,6 +1910,45 @@ def test_remove_field(self):
         self.assertEqual(definition[1], [])
         self.assertEqual(definition[2], {"model_name": "Pony", "name": "pink"})
 
+    def test_remove_field_non_nullable(self):
+        """
+        Tests the RemoveField operation with explicit default
+        """
+        project_state = self.set_up_test_model("test_rmflnn")
+        # Test the state alteration
+        operation = migrations.RemoveField("Pony", "weight")
+        new_state = project_state.clone()
+        Pony = project_state.apps.get_model("test_rmflnn.Pony")
+        Pony.objects.create(pink=1, weight=1337)
+        operation.state_forwards("test_rmflnn", new_state)
+
+        with connection.schema_editor() as editor:
+            operation.database_forwards("test_rmflnn", editor, project_state, new_state)
+
+        # And test reversal
+        # MySQL populates NOT NULL cols with implicit defaults instead of raising
+        # an error.
```

Comment:
Interesting I did not know this… I don't use MySQL but I'm hardly surprised 😛

I wonder whether this can be utilised in any way.